### ISSUE TITLE
Config launcher doesn't show native resolution in windowed mode

### ISSUE THREAD

**Issue by Sheepyhead** [Bouncyrock/TaleSpire-Alpha-Public-Issue-Tracker#32]
**Describe the bug**
When I start Talespire and switch to windowed mode, the resolution options no longer include my native resolution of 1920x1080, but the resolution one step lower.

**To Reproduce**
1. Open Talespire on a 1920x1080 monitor
2. In the config launcher check "windowed mode"

**Comment by Ro4m3r** [comment]
I don't think Unity actually uses true "full screen" even when windowed mode isn't checked, as it doesn't seem to take over the GPU entirely (alt-tabbing does not force the GPU to "re-load" windows).

If this is the case, "full screen" is actually native resolution borderless windowed mode.

**Comment by Sheepyhead** [comment]
If that's the case, the options should be clarified

**Comment by Ro4m3r** [comment]
The UI you see when launching the game isn't done by BouncyRock, it is a Unity specific, automatically created dialogue. 

Eventually this will most likely be removed in the future and replaced by "traditional" in-game video options.

**Comment by Sheepyhead** [comment]
I'm aware, however we can only report issues on the software we have in our hands, not a hypothetical future version of it 👍

**Comment by Ro4m3r** [comment]
I'm not sure if the issue shouldn't be closed then and turned into a feature request for ingame video settings, since they cannot modify the dialogue.

**Comment by Ro4m3r** [comment]
Makes sense. I guess I'm used to the "open source" style of issues. Cheers.
